doc/content/gateways/moko-mkgw2-lw/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "MOKOSMART MKGW2-LW LoRaWAN Gateway"
+description: ""
+---
+
+MOKOSMART's 8 channel MKGW2 gateway is embedded with Semtech’s high performance multi-channel transceiver SX1301/SX1257 and MTK platform. It is for indoor use and is easy to install.
+
+{{< figure src="mkgw2-lw.jpg" alt="mkgw2-lw" class="plain float" >}}
+
+## Requirements
+
+1. User account on {{% tts %}} with rights to create gateways.
+2. MOKOSMART MKGW2-LW connected to the internet via Ethernet or WiFi. (For Ethernet and WiFi settings, see sections 5 and 6 of the [user manual](https://www.mokosmart.com/lorawan-gateway-mkgw2-lw/))
```

Comment:
I don't see a user manual here, this is a product overview page

Reply:
@Fakerallen